### DISCUSSION TITLE
Next-Level Virtual Meetings with Circle: Secure, Scalable, and Customizable

### DISCUSSION THREAD

**Discussion by zerostack0** [ant-media/Ant-Media-Server#6350] [Announcements]
![Designer (1)](https://github.com/ant-media/Ant-Media-Server/assets/23143596/2367618a-02d9-483c-b9f5-8fcc16a5ee6f)

🚀 Join Us at Tech Talks! 🚀

🌟 Topic: Next-Level Virtual Meetings with Circle: Secure, Scalable, and Customizable

🗓️ Date: May 30th
🕒 Time: 3:00 PM GMT+3

We are excited to announce this week's Tech Talks event, where we will dive into the innovative Circle Conference application. Whether you're a seasoned professional or just starting out, this session is perfect for anyone looking to enhance their virtual conference experience.

🔍 What You'll Learn:

- Introduction to basic features of Circle
- Screenshare capabilities
- Background replacement
- Recording options
- Quick overview of additional features (layout, look and feel, language, etc.)
- Live load test: loading The Circle, pagination, removing the load
- Q&A session with our experts

📢 Why Attend?

- Stay ahead with the latest tech trends
- Network with like-minded professionals
- Gain practical insights to improve your virtual events

Don’t miss out on this opportunity to explore how Circle can transform your virtual meetings into secure, scalable, and customizable experiences.

🌐 Location: https://meet.antmedia.io/Conference/techtalks

🔗 Save the Date: Don't miss out! [Add to your calendar](https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=NGQ1dWI4MDQxNWJpbGJtY2JmcmhxZHY2OWEgY19kZjQ5YzExZWMyNDhhYWUxMjQ3ZTIzNTFlZjBkNzU1OGNlNDgxYzg4MDZjMWFmNjRmYjIzZTQyMzIzNjQ2ZTZkQGc&tmsrc=c_df49c11ec248aae1247e2351ef0d7558ce481c8806c1af64fb23e42323646e6d%40group.calendar.google.com)

Looking forward to seeing you there! 💻

**Comment by brixxnatt** [comment]
https://meet.antmedia.io/Conference/techtalks  link is not found ?!

**Reply by zerostack0** [reply]
Hi @brixxnatt 

Our team is preparing the server for the load test section and it will be reachable soon. It will be available during the event. (in 34 mins)

**Comment by zerostack0** [comment]
Thank you to everyone who participated in Tech Talks this week!
